core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java

```diff
@@ -165,10 +165,4 @@ public JSONObject updateCartItem(JSONObject item, String itemId, String custToke
 
 
 
-	
-
-
-
```

Comment:
@nitheshkumar-hcl  What is changes in this file?

Reply:
Committing the changes as per review comments

Reply:
Removed the spaces in the file